### PR TITLE
One Click Installer Callbacks

### PR DESCRIPTION
**Pull request by shalomRachapudi**

### CODE REVIEW

File: src/interfacemain.cpp [hunk 1]
```diff
@@ -41,6 +45,21 @@ int main( int argc, char *argv[] )
...
     }
+ 
+    // Progress callbacks
+    try {
+	static RpmCallbacks rpmCallbacks;
+	static SourceCallbacks sourceCallbacks;
+	static MediaCallbacks mediaCallbacks;
+    }
+    catch ( const Exception& e )  {
+	qDebug() << "Failed to initialize OCI callbacks.";
```
Comment (antlarr) [inline src/interfacemain.cpp]: If we're going to exit without telling anything to the user, let's at least make it a qFatal so it's printed on stderr

File: src/interfacemain.cpp [hunk 2]
```diff
@@ -41,6 +45,21 @@ int main( int argc, char *argv[] )
...
+	exit( 1 );
+    }
+    catch (...) {
+	qDebug() << "Failed to initialize OCI callbacks.";
```
Comment (antlarr) [inline src/interfacemain.cpp]: same here

File: src/interfacemain.cpp (hunk 1 above)
Comment (shalomRachapudi) [inline src/interfacemain.cpp]: Sure, okay.

**Comment by antlarr** [comment]
Some issues have already been noted that will be fixed in another PR, but for now, I'll approve this one

**Comment by shalomRachapudi** [comment]
Yes, thank you.

**Pull request merged**